packages/tempus/unit_test/Tempus_UnitTest_SolutionHistory.cpp

```diff
@@ -32,78 +30,84 @@ TEUCHOS_UNIT_TEST(SolutionHistory, Default_Construction)
-  auto icSolution = rcp_const_cast<Thyra::VectorBase<double> >(inArgsIC.get_x());
+  auto icSolution =
+      rcp_const_cast<Thyra::VectorBase<double> >(inArgsIC.get_x());
   auto icState = Tempus::createSolutionStateX<double>(icSolution);
   sh->addState(icState);
   sh->initialize();
   TEUCHOS_TEST_FOR_EXCEPT(!sh->isInitialized());
 
   // Test the get functions (i.e., defaults).
-  TEST_COMPARE ( sh->getNumStates(), ==, 1);
-  TEST_COMPARE ( sh->getInterpolator()->order(), ==, 0);
-  TEST_COMPARE ( sh->getStorageType(), ==, Tempus::STORAGE_TYPE_UNDO);
-  TEST_COMPARE ( sh->getStorageLimit(), ==, 2);
+  TEST_COMPARE(sh->getNumStates(), ==, 1);
+  TEST_COMPARE(sh->getInterpolator()->order(), ==, 0);
+  TEST_COMPARE(sh->getStorageType(), ==, Tempus::STORAGE_TYPE_UNDO);
+  TEST_COMPARE(sh->getStorageLimit(), ==, 2);
 
   // Test the set functions.
-  sh->setName ("Testing");                                              TEUCHOS_TEST_FOR_EXCEPT(!sh->isInitialized());
-  sh->setStorageType (Tempus::STORAGE_TYPE_STATIC);  sh->initialize();  TEUCHOS_TEST_FOR_EXCEPT(!sh->isInitialized());
-  sh->setStorageTypeString ("Static");               sh->initialize();  TEUCHOS_TEST_FOR_EXCEPT(!sh->isInitialized());
-  sh->setStorageLimit(99);                           sh->initialize();  TEUCHOS_TEST_FOR_EXCEPT(!sh->isInitialized());
-
-  TEST_COMPARE( sh->getName        ()     , ==, "Testing");
-  TEST_COMPARE( sh->getStorageType ()     , ==, Tempus::STORAGE_TYPE_STATIC);
-  TEST_COMPARE( sh->getStorageTypeString(), ==, "Static");
-  TEST_COMPARE( sh->getStorageLimit()     , ==, 99);
+  sh->setName("Testing");
+  TEUCHOS_TEST_FOR_EXCEPT(!sh->isInitialized());
+  sh->setStorageType(Tempus::STORAGE_TYPE_STATIC);
+  sh->initialize();
+  TEUCHOS_TEST_FOR_EXCEPT(!sh->isInitialized());
+  sh->setStorageTypeString("Static");
+  sh->initialize();
+  TEUCHOS_TEST_FOR_EXCEPT(!sh->isInitialized());
+  sh->setStorageLimit(99);
+  sh->initialize();
+  TEUCHOS_TEST_FOR_EXCEPT(!sh->isInitialized());
 
-  TEST_FLOATING_EQUALITY( sh->minTime(), 0.0, 1.0e-14);
-  TEST_FLOATING_EQUALITY( sh->maxTime(), 0.0, 1.0e-14);
+  TEST_COMPARE(sh->getName(), ==, "Testing");
+  TEST_COMPARE(sh->getStorageType(), ==, Tempus::STORAGE_TYPE_STATIC);
+  TEST_COMPARE(sh->getStorageTypeString(), ==, "Static");
+  TEST_COMPARE(sh->getStorageLimit(), ==, 99);
 
+  TEST_FLOATING_EQUALITY(sh->minTime(), 0.0, 1.0e-14);
+  TEST_FLOATING_EQUALITY(sh->maxTime(), 0.0, 1.0e-14);
 }
 
-
 // ************************************************************
 // ************************************************************
 TEUCHOS_UNIT_TEST(SolutionHistory, Full_Construction)
 {
   std::string name = "Unit Test";
 
-  auto history = rcp(new std::vector<RCP<Tempus::SolutionState<double> > >);;
-  auto model = rcp(new Tempus_Test::SinCosModel<double>());
+  auto history = rcp(new std::vector<RCP<Tempus::SolutionState<double> > >);
+  ;
+  auto model    = rcp(new Tempus_Test::SinCosModel<double>());
```

Comment:
this probably is not what you want clang format to do for extra semicolon

Reply:
Interesting ... There were about a dozen places where we had double semicolons (';;').  Not sure it is something that clang-format can handle "correctly" for us.  It is a benign statement, but clang-format at least pointed it out.  Removing them.   Can it get an approval?